community/basic_examples/hw_aware_synthesis.ipynb

```diff
@@ -0,0 +1,601 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    from classiq import (
</pre><p>You can use just <code>from classiq import *</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/116/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>